demos/k8s-demo/chart/demo-app-secretless/README.md

```diff
@@ -0,0 +1,152 @@
+# demo-app-secretless
+
+[The Secretless Broker](https://github.com/cyberark/secretless-broker) is a connection broker which relieves client applications of the need to directly handle secrets to target services such as databases, web services, SSH connections, or any other TCP-based service.
+
+## TL;DR;
+
+```bash
+$ helm install -f values.yaml .
```

Comment:
is this really the minimal command? they have to edit that file to have the required `application*` values in it, no?